static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -84,6 +86,26 @@ define([
         var ophanUrl = 'https://dashboard.ophan.co.uk/graph/breakdown?ab=' + this.config.test.id;
         this.getElem('ophan').innerHTML = '<a href="' + ophanUrl + '"">graph</a>';
 
+        if (window.abConfigSwitches.abShowHypothesesDashboard) {
+            var hypothesis;
+            var criteria = this.config.test.audienceCriteria ? true : false;
+            var description = this.config.test.description ? true : false;
+            var reason = this.config.test.reason ? true : false;
+            var success = this.config.test.successMeasure ? true : false;
+            var outcome = this.config.test.idealOutcome ? true : false;
```

Comment:
usually I see people just use the property directly as a boolean.  But I'm sure someone with more JS knowledge will be along soon.